salt/codespeed/config/django-settings.py.jinja

```diff
@@ -14,7 +14,7 @@ DATABASES = {
         "USER": "{{ db_user}}",
         "PASSWORD": "{{ db_pass }}",
         "OPTIONS": {
-            "sslmode": "verify-full",
+            "sslmode": "verify-ca",
```

Comment:
this is a local kludge and should be handled differently. for simplicity... I think we should probably just have the salt-master run a a postgres cluster for local dev so we don't have to fire up yet another box.